pkg/local_workflows/json_schemas/data_test_summary.go

```diff
@@ -0,0 +1,51 @@
+package json_schemas
+
+const TestSummarySchema = `{
```

Comment:
What options might be available to us to further integrate this schema with the CLI codebase? Wondering specifically about:

- Generating Go "wire-format" structure types from the schema (so it becomes SSoT)
- Validating responses against it, at runtime if it's performant enough, in tests or in "strict mode" otherwise

Reply:
I think this introduces the opportunities to set those up but I will be honest I need to do some more research on the tooling options here. Both of those sound interesting but having some types is a step in the right direction. 

A quick search surfaces these:
* https://github.com/omissis/go-jsonschema
* https://pkg.go.dev/github.com/santhosh-tekuri/jsonschema/v5

One opportunity for runtime validation could be here:
https://github.com/snyk/cli/pull/5157/files#diff-9c2017eb57391e3dc8da9d6c7af7df0ee0fdb23aa039485a3876179d37070a08R195-R197

Reply:
question: instead of validating responses at runtime, would it be better/sufficient to couple the schema to a workflow test? i.e. a test to validate a workflow output against its schema.

this way we can be sure that the schema matches the output and so the schema can be used as the SSOT/contract to hand over to dependencies.

Reply:
> question: instead of validating responses at runtime, would it be better/sufficient to couple the schema to a workflow test? i.e. a test to validate a workflow output against its schema.

I think we will not be able to not validate the schema, just due to the plug and play nature with extensions. When consuming the data for example in the output module we want to ensure that the data has the format as expected.